packages/server/src/service-worker.ts

```diff
@@ -6,7 +6,7 @@ const CACHE = 'precache';
 /**
  * Communication channel for drive access
  */
-const broadcast = new BroadcastChannel('/api/drive.v1');
+const broadcast = new BroadcastChannel('/sw-api.v1');
```

Comment:
Maybe worth also updating the dosctring above, and any other mentions of `drive` to something a bit more generic?